src/content/docs/en/guides/markdown-content.mdx

```diff
@@ -119,6 +119,8 @@ I can link internally to [my conclusion](#conclusion) on the same page when writ
 I can use the URL `https://example.com/page-1/#introduction` to navigate directly to my Introduction on the page. 
 ```
 
+If you want to link to a heading from an anchor link, Astro generates `id`s based on [github-slugger](https://github.com/Flet/github-slugger), you can find more examples in [the github-slugger documentation](https://github.com/Flet/github-slugger?tab=readme-ov-file#usage).
```

Comment:
LGTM! Maybe one suggestion to avoid two links given they’re basically to the same thing, what do you think? Oh, and I also ended up deleting the first part given right above the code block we mention “you can link directly…”:

```suggestion
Astro generates `id`s based on `github-slugger`. You can find more examples in [the github-slugger documentation](https://github.com/Flet/github-slugger#usage).
```

Reply:
Great idea!